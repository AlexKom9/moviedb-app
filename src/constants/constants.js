//movies
export const UPDATE_MOVIES = "UPDATE_MOVIES";
export const CHANGE_PAGE = "CHANGE_PAGE";
export const FETCHING_MOVIES = "FETCHING_MOVIES";
export const UPDATE_FILTERS = "UPDATE_FILTERS";
export const RESET_FILTERS = "RESET_FILTERS";
export const ERROR_GET_MOVIES = "ERROR_GET_MOVIES";


//Authentication
export const UPDATE_AUTH = "UPDATE_AUTH";
export const LOG_OUT = "LOG_OUT";

//Account

export const UPDATE_FAVORITE = "UPDATE_FAVORITE";
export const UPDATE_WATCHLIST = "UPDATE_WATCHLIST";

//Login form
export const TOGGLE_LOGIN_FORM = "TOGGLE_LOGIN_FORM";
export const HIDE_LOGIN_FORM = "HIDE_LOGIN_FORM";






