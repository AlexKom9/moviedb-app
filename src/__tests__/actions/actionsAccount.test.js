import {
  actionCreatorUpdateFavorite,
  actionCreatorUpdateWatchlist
} from "../../actions/actionsAccount";
import * as constants from "../../constants/constants";

describe("actionsAccount", () => {
  test("actionCreatorUpdateAuth", () => {
    const data = [];

    const result = actionCreatorUpdateFavorite(data);
    // expect(result.type).toBe(constants.UPDATE_AUTH);
    // expect(result.payload.user).toBe(payload.user);
    // expect(result.payload.session_id).toBe(payload.session_id);
    // expect(result.payload.isAuth).toBe(payload.isAuth);
    expect(result).toMatchSnapshot();
  });
});


