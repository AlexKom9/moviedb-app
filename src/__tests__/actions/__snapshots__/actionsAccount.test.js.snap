// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actionsAccount actionCreatorUpdateAuth 1`] = `
Object {
  "payload": undefined,
  "type": "UPDATE_FAVORITE",
}
`;

exports[`actionsAuthentication actionCreatorUpdateAuth 1`] = `
Object {
  "payload": Object {
    "session_id": 1,
    "user": 1,
  },
  "type": "UPDATE_AUTH",
}
`;
